test/functional/apps/home/_navigation.ts

```diff
@@ -105,7 +105,7 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
       await browser.get(`${basePath}/app/kibana#/home`, false);
       await retry.waitFor(
         'navigation to home app',
-        async () => (await browser.getCurrentUrl()) === `${basePath}/app/kibana#/home?_g=()`
+        async () => (await browser.getCurrentUrl()) === `${basePath}/app/kibana#/home`
```

Comment:
Interesting side effect of decoupling `unhashUrl` util from `State` objects. 
Apparently previously some where here:
https://github.com/elastic/kibana/blob/d92965ab2110be8bfdb4d71dbb6952d297a69f18/src/legacy/ui/public/chrome/api/sub_url_hooks.js#L34
If there was no state query param in url it is still fallback to default `{}` and it synced it back as `_g=()`. 
Now this isn't happening 

This doesn't seem to be breaking change to me or a ux degradation. Just 2 e2e tests relied on it as on implementation details. 
Let's discuss if this is a problem or not